mapswipe_workers/tests/unittests/fixtures/build_area/project_drafts_new.json

```diff
@@ -0,0 +1,104 @@
+[
```

Comment:
okay, now I'm confused. What is the difference of this file and the file `project_drafts.json`. Are both needed?